.travis.yml

```diff
@@ -3,8 +3,8 @@ jdk: oraclejdk8
 
 android:
   components:
-    - android-26
-    - build-tools-26.0.1
+    - android-27
+    - build-tools-27.0.3
```

Comment:
As of merging this PR, Travis builds fails due to Android-27 licensing issues. A temporary workaround fixes this in https://github.com/vestrel00/android-dagger-butterknife-mvp/commit/303c65c93080c0822c8474a7a6c0045ce01e22ae.